lib/app/data/providers/secure_storage_provider.dart

```diff
@@ -177,6 +177,21 @@ class SecureStorageProvider {
     );
   }
 
+  Future<void> writeTimerStartTimeInSeconds({
+    required int timerStartTimeInSeconds,
+  }) async {
+    await _secureStorage.write(
```

Comment:
Let's move this to Get Storage since we already have it integrated in the application now